src/main/scala/org/apache/spark/deploy/armada/submit/ArmadaClientApplication.scala

```diff
@@ -306,14 +311,86 @@ private[spark] class ArmadaClientApplication extends SparkApplication {
     ()
   }
 
+  private def getExecutorContainers(numExecutors: Int, driverServiceName: String, conf: SparkConf): Seq[Container] = {
+    return (for (i <- 0 until numExecutors)
+      yield getExecutorContainer(i, driverServiceName, conf))
```

Comment:
This seems clearer and more idiomatic to me:
```
    (0 until numExecutors).map
      {getExecutorContainer(_, driverServiceName, conf)}
```

Reply:
addressed in f0f7ab8